src/consts/consts.py

```diff
@@ -125,7 +125,6 @@ def all(cls):
 WORKER_TF_CPU_MODE = HOST_PASSTHROUGH_CPU_MODE
 NUMBER_OF_MASTERS = 3
 NAMESPACE_POOL_SIZE = 15
-PODMAN_FLAGS = "--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file"
```

Comment:
I just saw `--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file` is hard-coded in https://github.com/openshift/assisted-test-infra/blob/c5d5a2867a6f552d250e39e0b3513c8fdfd33750/src/assisted_test_infra/test_infra/controllers/tang_controller/tang_controller.py#L38